index.js

```diff
@@ -35,7 +35,7 @@ function fastifyRedis (fastify, options, next) {
     fastify.redis[namespace] = client
   } else {
     if (fastify.redis) {
-      next(new Error('fastify-redis has already been registered'))
+      return next(new Error('fastify-redis has already been registered'))
```

Comment:
this is a fix.

In order to have a good (automatic) changelog in the GH release, could you move this line in another pr please?

Reply:
Done ^^.